components/org.wso2.finance.open.banking.conformance.mgt/pom.xml

```diff
@@ -39,6 +39,18 @@
             <artifactId>gson</artifactId>
             <version>${com.google.code.gson.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>1.4.196</version>
```

Comment:
take these version values to the root pom

Reply:
done